lib/internal/bootstrap_node.js

```diff
@@ -78,11 +78,13 @@
         NativeModule.require('_third_party_main');
       });
 
-    } else if (process.argv[1] === 'debug') {
-      // Start the debugger agent
-      NativeModule.require('_debugger').start();
+    } else if (process.argv[1] === 'inspect' || process.argv[1] === 'debug') {
+      if (process.argv[1] === 'debug') {
+        process.emitWarning(
+          '`node debug` is deprecated. Please use `node inspect` instead.',
+          'DeprecationWarning', 'DEP0062');
```

Comment:
It is a bit of a technicality but`node debug` used to produce `DEP0062` before this change. There is no other place in the code that can produce DEP0062 anymore as `--debug` is no longer an option. I think it is simpler to continue to use DEP0062 with an updated message. @jasnell thoughts? How do I reserve a new deprecation message, if there is a preference for that?